tests/core/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
It'd be worth sticking to npm/nodejs for all the building than a mix of npm/bash. The package.json is a good self-documenter for the commands that can be run; and nodejs would be more maintainable, assuming this package will scale up. In this case it's fine, but worth settling on it early. It's something I'd be happy to add as a separate PR, if you think it's a good idea.

Eg:

*build.ts*
```
...
function getTestFiles() ...
async function compileTestFile(...) { exec('npm run asc {...}') }

Promise.all(getTestFiles().map(compileTestFile))
```

*package.json*
```
"scripts": {
    "build": "ts-node build.ts"
}
```

Reply:
Yeah, I like it. Feel free to make a PR with that change!